source/common/logger/ur_logger.hpp

```diff
@@ -11,31 +11,41 @@
 #include "ur_util.hpp"
 
 namespace logger {
-extern Logger logger;
+
+Logger create_logger(std::string logger_name);
+
+inline Logger &get_logger(std::string name = "") {
```

Comment:
This means that tests and other places you log without explicit init will print `<>[INFO]: message`. Maybe instead we can use 'common' ?